images/django/Dockerfile

```diff
@@ -0,0 +1,57 @@
+#####
+## Base image for all images that run Django apps
+##
+## Expects a previous stage with the alias "python-build" containing pre-built wheels
+## for the application in the /build/wheelhouse directory
+#####
+
+ARG ESGF_REPOSITORY_BASE=esgfdeploy
+ARG ESGF_IMAGES_VERSION=latest
+FROM ${ESGF_REPOSITORY_BASE}/conda:${ESGF_IMAGES_VERSION}
+
+USER root
+
+# Install gunicorn
+RUN pip install --no-cache-dir 'gunicorn==20.0.4'
+
+# Install gunicorn config file
+COPY gunicorn.conf.py /etc/gunicorn/conf.py
+COPY wsgi-serve.sh /usr/local/bin/
+
+# Install whitenoise to handle static files and django-flexi-settings
+# for smart handling of settings
+RUN pip install --no-cache-dir \
```

Comment:
Merge this with the gunicorn installation above to reduce number of layers.

Reply:
@mkjpryor-stfc The only thing missing from base-images seemed to be the pre/post install hooks from the python-build and django images, so I made that change. And the other two things are now fixed. Thanks!